maxim/test_runs/run_utils.py

```
@@ -21,14 +21,14 @@ def get_variables_from_row(
     for column_name, column_type in data_structure.items():
         if column_type == "FILE_URL_VARIABLE":
             variables[column_name] = Variable(
-                type_="file", payload={"files": [{"url": row[column_name]}]}
+                type="file", payload={"files": [{"url": row[column_name]}]}
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)